gulpfile.js/bpk-component-icon/gulpfile.babel.js

```diff
@@ -26,4 +27,6 @@ gulp.task('copy', () =>
     .pipe(gulp.dest('./packages/bpk-component-icon')),
 );
 
-gulp.task('generateIcons', gulp.series('copy'));
+gulp.task('clean', () => Promise.all([rimraf('./packages/bpk-component-icon/sm'),
```

Comment:
Any reason we can't just do this as a command in the package file instead of introducing a whole new library for this one part?

Reply:
Similar to the `npm run clean` command we have already

Reply:
I've removed the dep, replaced with native `fs`, I think keeping things out of `package.json` is an advantage when we already utilise `gulp` for this.